include/caffe/layers/cudnn_conv_layer.hpp

```diff
@@ -93,6 +94,9 @@ class CuDNNConvolutionLayer : public ConvolutionLayer<Dtype> {
                    const vector<Blob<Dtype>*>& top,
                    const size_t workspace_bytes);
 
+  size_t ComputeWorkspaceSize(const vector<Blob<Dtype>*>& bottom,
```

Comment:
I would rename this to e.g.`ComputeFindExWorkspaceSize` because it only computes the workspace size of FindEx. The workspace size of the actual algorithms is set differently.